### PR TITLE
Add BRL system log event info

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Added burst rate limit system log event info
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-467168](https://oktainc.atlassian.net/browse/OKTA-467168)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ The following org-based System Log events record system events related to your o
 * [`system.org.rate_limit.warning`](/docs/reference/api/event-types/?q=system.org.rate_limit.warning)<br>
 This event is sent when an endpoint is nearing its rate limit.
 
+* [`system.org.rate_limit.burst`][/docs/reference/api/event-types/?q=system.org.rate_limit.burst]<br>
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md:17]:
```suggestion
* [`system.org.rate_limit.burst`](/docs/reference/api/event-types/?q=system.org.rate_limit.burst)<br>
```

**Review by brianduffield-okta — APPROVED** [review]
One fix to link required, otherwise, LGTM

**Review by joec-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Okta provides rate limits for orgs based on the traffic that they expect to have
...
 
-Additionally, burst rate limits typically apply on top of any rate limit increase that an org may have, such as [DyanmicScale](/docs/reference/rl-dynamic-scale/). For example, the default limit on `/api/v1/authn` is 600 requests per minute. If an org is expecting traffic to require 6000 requests per minute, the org would purchase DynamicScale 10x. The burst rate limit in this scenario would provide 5x coverage on top of the 6000 and ensure peace of mind for any unplanned spike in use.
+Additionally, burst rate limits typically apply on top of any rate limit increase that an org may have, such as [DynamicScale](/docs/reference/rl-dynamic-scale/). For example, the default limit on `/api/v1/authn` is 600 requests per minute. If an org is expecting traffic to require 6000 requests per minute, the org would purchase DynamicScale 10x. The burst rate limit in this scenario would provide 5x coverage on top of the 6000 and ensure peace of mind for any unplanned spike in use.
```
[the remark this answers is not in the document]
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:41]: I'll communicate this to Janet in my hand-off for a team discussion.

**Pull request merged**